docs/documenting/installation-manual.md

```diff
@@ -0,0 +1,35 @@
+# Twomes Measurement Device Installation Manual
+
+The purpose of a device installation manual is to help users physically install and power up devices of your specific measurement device type. The contents of the folder [/docs/installationmanual/](/docs/installationmanual/) of a Twomes measurement device repository can be used as a template for this. 
```

Comment:
> The contents of the folder [/docs/installationmanual/](/docs/installationmanual/) of a Twomes measurement device repository can be used as a template for this.

@henriterhofte Do you want to link to a specific file in a repository? If not, I suggest changing this to the following:
```suggestion
The purpose of a device installation manual is to help users physically install and power up devices of your specific measurement device type. The contents of the folder `/docs/installationmanual/` of a Twomes measurement device repository can be used as a template for this. 
```